src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/web-application-firewall.mdx

```diff
@@ -14,7 +14,11 @@ WAF operates in the seventh layer of the OSI model, the application layer, where
 
 **Web Application Firewall** is based on requests *scoring* methodology. Each HTTP/HTTPS request is compared to a very strict and detailed set of application standards and given a *score* that is associated with a particular family of threats. According to the *score* received by the request, it can be released or blocked directly in Azion's edge nodes, before the threat reaches its origin or causes any type of damage. You define the desired level of sensitivity for blocking each family of threats.
 
-To avoid blocking lawful requests and malfunctions of your application, you must perform a learning step. In this step, WAF identifies the legitimate behaviors of your application by placing them on an *allowlist*. If internal traffic, tests, and false positives are being blocked by WAF, you can also fine-tune its settings in the **Tuning** tab, inside a created WAF.
+WAF can operate in two modes: **Learning** or **Blocking**. You can set the mode in a rule in [Rules Engine](/en/documentation/products/secure/edge-firewall/rules-engine/) with the **Set WAF Rule Set** behavior or via [Azion API](https://api.azion.com/#cee49f12-e1d1-4d93-b117-2ef4554121b6).
+
+You can use the **Learning** mode to avoid blocking lawful requests and malfunctions of your application. In this step, WAF identifies the legitimate behaviors of your application by placing them on an *allowlist*.
```

Comment:
```suggestion
You can use the **Learning** mode to avoid blocking lawful requests and malfunctions of your application. In this step, WAF identifies the legitimate behaviors of your application by placing them in an *allowlist*.
```